lib/core/src/packStruct.cpp

```diff
@@ -2704,18 +2691,15 @@ unpackPointerItem( packItem_t &myPackedItem, const void *&inPtr,
     }
     case PACK_INT_TYPE:
         if ( myDim == 0 ) {
-            outPtr = addPointerToPackedOut( unpackedOutput,
-                                            numElement * elementSz, NULL );
-            status = unpackIntToOutPtr( inPtr, outPtr, numElement,
-                                        myPackedItem.name, irodsProt );
+            outPtr = addPointerToPackedOut( unpackedOutput, numElement * elementSz, NULL );
+            unpackIntToOutPtr( inPtr, outPtr, numElement, myPackedItem.name, irodsProt );
```

Comment:
This line now mirrors the master branch. Is it a bug in master that the result is not captured in `status`? I'm not sure which implementation is the correct one, but capturing the result in `status` does feel more correct.